wolfssl/wolfcrypt/port/Espressif/esp32-crypt.h

```diff
@@ -42,54 +44,76 @@
  * WC_HW_WAIT_E - waited too long for HW, fall back to SW
  */
 
-/* exit code only used in Espressif port */
+/* Exit codes only used in Espressif port: */
+enum {
+    ESP_MP_HW_FALLBACK          = (WC_LAST_E - 2),
+    ESP_MP_HW_VALIDATION_ACTIVE = (WC_LAST_E - 3)
+};
 
 /* MP_HW_FALLBACK: signal to caller to fall back to SW for math:
  *   algorithm not supported in SW
  *   known state needing only SW, (e.g. ctx copy)
  *   any other reason to force SW */
-#define MP_HW_FALLBACK (-108)
+#define MP_HW_FALLBACK ESP_MP_HW_FALLBACK  /* was -108 */
```

Comment:
Having comments in macro definitions can have unusual unintended consequences. Recommend instead putting it into the comment block above.

Reply:
Good point on the macro comments. I've updated those and adjusted some line lengths.